Potapov Vladislav/task2.js

```diff
@@ -0,0 +1,11 @@
+function hashToArray(hash) {
+    return Object.entries(hash)
```

Comment:
@TheUnforgivenOne According to the main task we shouldn't use es6, so please don't use `Object entries`